packages/cli/tests/images/build.js

```diff
@@ -32,7 +32,7 @@ exports.default = exports.full = Object.assign({}, common, {
 
 exports.sass = `
 <body>
-	<div class="background__21gOq">
+	<div class="background__16qjh">
```

Comment:
Let's revert this bad boy, the Travis version expects `21g...` we'll make better tests soon!

Reply:
alright